hubstorage/resourcetype.py

```diff
@@ -29,6 +47,9 @@ def _iter_lines(self, _path, **kwargs):
         return r.iter_lines()
 
     def apirequest(self, _path=None, **kwargs):
+        if self._supports_msgpack and kwargs.get('method') == 'GET':
```

Comment:
it's possible to get `method='get'` or `method='gEt'` here, it's converted to upper only in `requests` lib